getting_started/hello_world_bash.Rmd

```diff
@@ -1,14 +1,31 @@
 ---
-title: "Hello World"
+title: "Hello World (Bash)"
 parent: Getting started
-nav_order: 2
+nav_order: 3
 ---
 
+```{r init, include=FALSE}
+library(tidyverse)
+
+source("../src/helper/md_helpers.R")
+
+# change wd to where the files are located
+knitr::opts_knit$set(root.dir = "../examples/hello_world/") 
+# Remove hashtags before bash output
+knitr::opts_chunk$set(comment = NA)
+```
 
 # Hello World
 
 We have provided a simple **Hello World** component as an introduction to viash. Follow the steps below to learn how to run a component and get a result back from it.
 
+## Prerequisites
+
+To follow along with this tutorial, you need to have this software installed on your machine:
+
+- An [installation of viash](/getting_started/installation).
+- A **Unix shell** like Bash or Zsh.
```

Comment:
@BlackDragonBE We haven't tested whether viash works with Zsh, so maybe it's best to simply mention Bash as a requirement.

Reply:
Replaced this particular sentence everywhere with just the Bash Unix shell.